docs/src/app/components/pages/components/Paper/ExampleCircle.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Paper from 'material-ui/lib/paper';
+
+const style = {
+  height: 100,
+  width: 100,
+  marginLeft: 40,
```

Comment:
What about `margin: 20` instead to fix
![capture d ecran 2016-01-06 a 16 44 30](https://cloud.githubusercontent.com/assets/3165635/12146601/cccf67f8-b494-11e5-95da-c03608f2c0f5.png)